.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Report a problem for the OPA community to investigate
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+<!-- Thanks for opening an issue to request a feature or file a bug!
+If you provide some basic information, it helps us address problems faster. -->
+
+## Short description 
```

Comment:
It would be good to include a checklist of common information like we have today: https://raw.githubusercontent.com/open-policy-agent/opa/main/.github/ISSUE_TEMPLATE.md